packages/make-styles-webpack-loader/src/webpackLoader.test.ts

```diff
@@ -0,0 +1,168 @@
+import * as fs from 'fs';
+import { createFsFromVolume, Volume } from 'memfs';
```

Comment:
is this standard lib needed to write tests for webpack loader ?

Reply:
> Pay close attention to this next step as we'll be using the Node.js API and memfs to execute webpack. This lets us avoid emitting output to disk and will give us access to the stats data which we can use to grab our transformed module:

https://webpack.js.org/contribute/writing-a-loader/#testing